page-hours-json.php

```diff
@@ -405,11 +405,17 @@
     
     <div class="locations-more">
       <h3>Locations without hours</h3>
+	  <!-- DIRC -->
       <h4><a href="/dirc">Digital Instruction Resource Center (DIRC)</a></h4>
-      <a href="/locations/#!digital-instruction-resource-center-dirc">Map:&nbsp; 14N-132</a>
+      <p><a href="/locations/#!digital-instruction-resource-center-dirc">Map:&nbsp; 14N-132</a></p>
+	  <!-- STATA -->
       <h4>Stata Center Book Drop</h4>
-      <a href="/locations/#!stata">Map:&nbsp;  32-Student Street</a> </div>
-  </div>
+      <p><a href="/locations/#!stata">Map:&nbsp;  32-Student Street</a></p>
+	  <!-- Physics Reading Room -->
+	  <h4>Physics Reading Room</h4>
+	  <p>Building 4-332 | 617.253.1791<br>
+      <a href="mailto:jytqtran@mit.edu">Tan-Quy Tran</a>, Reading Room Assistant</p>
+	</div>
```

Comment:
Great eye!

On Fri, Jun 17, 2016 at 9:26 AM frances botsford <notifications@github.com>
wrote:

> In page-hours-json.php
> <https://github.com/MITLibraries/MITlibraries-parent/pull/133#discussion_r67509424>
> :
>
> >        <h4>Stata Center Book Drop</h4>
> > -      <a href="/locations/#!stata">Map:
>
> &nbsp;  32-Student Street</a> </div>
> > -  </div>
> > +      <p><a href="/locations/#!stata">Map:&nbsp;  32-Student Street</a></p>
> > +	  <!-- Physics Reading Room -->
> > +	  <h4>Physics Reading Room</h4>
> > +	  <p>Building 4-332 | 617.253.1791<br>
> > +      <a href="mailto:jytqtran@mit.edu">Tan-Quy Tran</a>, Reading Room Assistant</p>
> > +	</div>
>
> Is there a missing close div now? i see the comment below says it's
> closing the .content div but this close is for locations-more
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/MITLibraries/MITlibraries-parent/pull/133/files/8b10ced0319d7fe8898baf3d3c01909e276d3161#r67509424>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABv1VSjYCa6Dd3gRRh858e4CzaFX2cfHks5qMqCVgaJpZM4I2eNt>
> .
>